src/pages/quiz-page/quiz-page.component.tsx

```diff
@@ -0,0 +1,91 @@
+import { useForm } from 'react-hook-form';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { ROUTES } from '../../shared/constants/routes.constants';
+import { Logo } from '../../shared/logo/logo.component';
+import { Select } from './components/select-component/select.component';
+import { Radio } from './components/radio-component/radio.component';
+import { RootState, Data } from './quiz-page.intarface';
+import { quizAction } from '../../store/quiz/actions/quiz.action';
+import { fetchPetsAsync } from '../../store/result/actions/result.actions';
+import styles from './quiz-page.module.scss';
+
+import { SliderSection } from '../filter/components/slider-section/slider-section.component';
+
+import dog from '../../images/dog-quiz.svg';
+
+export const QuizPage: React.FC = () => {
+  const { register, handleSubmit } = useForm<Data>();
+
+  const dispatch = useDispatch();
+
+  const onSubmit = (data: Data): void => {
+    dispatch(quizAction(data));
+  };
+
+  useEffect(() => {
+    dispatch(fetchPetsAsync.request());
+  }, []);
+
+  const pets = useSelector((state: RootState) => state.result.resultStore);
+  const dogs: any = [];
+  pets.forEach(dog => {
+    dogs.push(dog.breed);
+  });
+
+  const deltaPositionWeight = 0.0093;
```

Comment:
Probably you will  need to go back to this element, beacouse I changed these settings in filter page - so maybe you will have to change them too.
But I'll let you know if you have to.